src/scanner.cc

```diff
@@ -844,8 +844,11 @@ struct Scanner {
     }
 
     // Open delimiters for literals
-    if (valid_symbols[IDENTIFIER_HASH_KEY] && iswalpha(lexer->lookahead)) {
-      while (iswalpha(lexer->lookahead)) advance(lexer);
+    if (valid_symbols[IDENTIFIER_HASH_KEY]
+        && (iswalnum(lexer->lookahead) || lexer->lookahead == '_')) {
```

Comment:
I think the first character can't be a number, so this first check may need to be `iswalpha`, not `iswalnum`.

Reply:
oh, right

Reply:
I pushed this change